src/test/index.ts

```diff
@@ -0,0 +1,20 @@
+// This file is used by VS Code's default test runner to configure Mocha before the test run.
+
+/* tslint:disable:no-var-keyword no-var-requires */
+var testRunner = require("vscode/lib/testrunner");
+/* tslint:enable:no-var-keyword no-var-requires */
+
+let mochaOptions: any = {
+    ui: "tdd",
```

Comment:
Is there much of a difference between "tdd" and "bdd"? I see that you are using `suite` and `test` rather than `describe` and `it`, are they functionally the same?